src/main/reactjs/src/components/clerkTranslator/overview/ClerkTranslatorDetailsField.tsx

```diff
@@ -37,12 +37,14 @@ const getFieldError = (
 export const ClerkTranslatorDetailsField = ({
   translator,
   field,
-  onChange,
+  onFieldChange,
   ...rest
 }: {
   translator: ClerkTranslator | undefined;
   field: keyof ClerkTranslatorBasicInformation;
-  onChange: (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>) => void;
+  onFieldChange: (
+    e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => void;
 } & CustomTextFieldProps &
   TextFieldProps) => {
```

Comment:
Tälle voisi varmaan tehdä oman interfacen, joka extendaa CustomTextFieldPropsin. CustomTextFieldillakin voisi olla oma interface, joka extendaa TextFieldPropsin

Reply:
Jep, hyvä idea.

Reply:
Hmm, ilmeisesti ei mene ainakaan täysin helposti: TextFieldPropsin extendaaminen aiheuttaa virheen `An interface can only extend an object type or intersection of object types with statically known members.`.

Tämä siis johtunee siitä että TextFieldProps on type alias unionille `StandardTextFieldProps | FilledTextFieldProps | OutlinedTextFieldProps`.

Toisaalta muuttaminen tyypiksi eli allaoleva toimii: 
```export type CustomTextFieldProps = {
  showHelperText?: boolean;
} & TextFieldProps;
```

Reply:
joo, näyttää siltä että TextFieldProps on type(yleensä ne on interface) ja sen takia se ei onnistu. toi olle olevakin näyttää hyvältä. Toinen vaihtoehto käyttää jotain tiettyä TextfieldPropsia kuten OutlinedTextFieldProps

```
interface CustomTextFieldProps2 extends OutlinedTextFieldProps {
  property: string;
}
```

mutta tämä ehkä saattaa aiheuttaa ongelmia :)

Reply:
Niin, mieluummin olisi lyömättä lukkoon sitä TextFieldin varianttia. Tein tuolla tyyppimäärittelyllä.